packages/material-ui/src/Modal/manageAriaHidden.js

```diff
@@ -20,14 +20,14 @@ export function ariaHidden(show, node) {
   if (show) {
     node.setAttribute('aria-hidden', 'true');
   } else {
-    node.removeAttribute('aria-hidden');
+    node.setAttribute('aria-hidden', 'false');
```

Comment:
Was this necessary for the fix? If not I'd rather not change this.

> When the element is presented, authors MUST set the aria-hidden attribute to false or remove the attribute [...]
> --- [W3 WAI-ARIA](https://www.w3.org/WAI/PF/aria/states_and_properties#aria-hidden)

Seems like it shouldn't matter.

Reply:
It was in the 'Expected Behavior' of the issue